restapi.go

```diff
@@ -2874,6 +2874,76 @@ func (s *Session) FollowupMessageDelete(appID string, interaction *Interaction,
 	return s.WebhookMessageDelete(appID, interaction.Token, messageID)
 }
 
+// ------------------------------------------------------------------------------------------------
+// Functions specific to stage instances
+// ------------------------------------------------------------------------------------------------
+
+// StageInstanceData is provided to StageInstanceCreate
+type StageInstanceData struct {
+	// ChannelID represents the id of the Stage channel
+	ChannelID string `json:"channel_id"`
+	// Topic of the Stage instance (1-120 characters)
+	Topic string `json:"topic"`
+	// PrivacyLevel of the Stage instance (default GUILD_ONLY)
+	PrivacyLevel *StageInstancePrivacyLevel `json:"privacy_level,omitempty"`
+	// SendStartNotification will notify @everyone that a Stage instance has started
+	SendStartNotification *bool `json:"send_start_notification,omitempty"`
+}
+
+// StageInstanceCreate will creates a new Stage instance associated to a Stage channel.
+// Returns that Stage instance created
+func (s *Session) StageInstanceCreate(data *StageInstanceData) (si *StageInstance, err error) {
+	body, err := s.RequestWithBucketID("POST", EndpointStageInstances, data, EndpointStageInstances)
+	if err != nil {
+		return
+	}
+
+	err = unmarshal(body, &si)
+	return
```

Comment:
```suggestion
	return unmarshal(body, &si)
```

Reply:
Following the convention of restapi.go

Assign an Issue / another PR to makes this changes